Remove key 'license' to avoid error `project.license` must be valid exactly by one definition
author: Loris Bennett (FU Berlin)
diff -ruN whatshap-2.6.orig/pyproject.toml whatshap-2.6/pyproject.toml
--- whatshap-2.6.orig/pyproject.toml	2025-04-11 18:20:54.000000000 +0200
+++ whatshap-2.6/pyproject.toml	2025-05-19 13:00:41.787071949 +0200
@@ -9,7 +9,6 @@
 ]
 description = "phase genomic variants using DNA sequencing reads"
 readme = "doc/README.rst"
-license = "MIT"
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
